cert-manager-multi-arch.md

```diff
@@ -0,0 +1,728 @@
+
+```shell
+$ oc create configmap issued-ca-bundle \
+--from-file=ca-bundle.crt=./ca.crt \
+-n openshift-config
+```
+
+3. Use the created config map and update the proxy configuration across the entire cluster.
+
+```shell
+$ oc patch proxy cluster \
+--type=merge \
+--patch='{"spec":{"trustedCA":{"name":"issued-ca-bundle"}}}'
+```
+
+
+Wait a bit for the changes to propagate through your cluster, and ensure its health using fundamental commands like:
+```shell
+$ oc get nodes
+$ oc get pods --all-namespaces
+$ oc get clusteroperators
+```
+
+Fantastic! Your cluster is now fully configured to fortify all connections with the bespoke custom certificate.
+
+
+
+### See it in action
+Now, verify if your connections are indeed being served by the certificates.
+
+> Note: chage the `<cluster-domain>` with the base domain name for your cluster.
+
+1. 
+```shell
+$ echo Q | openssl s_client -connect console-openshift-console.apps.<cluster-domain>:443 -showcerts 2>/dev/null | openssl x509 -noout -subject -issuer -enddate
+```
+
+Sample result:
+```shell
+subject=CN = *.apps.<cluster-domain>  # <- check this
+issuer=CN = selfsigned-root-ca        # <- check this
+notAfter=Apr  2 13:15:59 2024 GMT
+```
+
+2. 
+```shell
+$ curl -v --cacert ./ca.crt https://console-openshift-console.apps.<cluster-domain>
+```
+
+Sample result:
+```shell
+*   Trying 20.111.222.333:443...
+* Connected to console-openshift-console.apps.<cluster-domain> (20.111.222.333) port 443 (#0)
+...
+*  CAfile: ./ca.crt
+...
+* Server certificate:
+*  subject: CN=*.apps.<cluster-domain>
+*  start date: Jan  3 13:15:59 2024 GMT
+*  expire date: Apr  2 13:15:59 2024 GMT
+...
+*  issuer: CN=selfsigned-root-ca
+*  SSL certificate verify ok.      # <- check this
+...
+```
+Verify SSL authentication, cross-check both the `CN` (common name) and certificate start and expiration details. Ensure alignment with expectations, and rest assured, cert-manager will handle automatic certificate renewal :repeat: :dart: 
+
+
+## Wrap Up
+
+In this article, I walked you through the intricacies of managing certificates on OpenShift, emphasizing its compatibility across diverse architectures with the enhanced support of the cert-manager Operator in `v1.13.0`.
+
+From deploying an OpenShift cluster on IBM Power Virtual Server (VS) to installing the cert-manager Operator, we explored a seamless end-to-end process. The practical side covered creating a Self-Signed CA Issuer, issuing a root certificate, and utilizing cert-manager for managing certificates in the default ingress controller for secure applications.
+
+Awesome! I hope you found this guide both informative and actionable. If you have any questions or insights, feel free to share them in the comments. Happy coding! :rocket: :star2: 
+
+And as promised, here are a few useful links for further reference:
+
+## Useful links
+- https://docs.openshift.com/container-platform/4.14/security/cert_manager_operator/index.html
```

Comment:
Consider changing links `/4.14/` into `/latest/`, so those links automatically redirect you to latest OpenShift docs. WDYT